src/ContextualPopup/ContextualPopup.js

```diff
@@ -222,6 +229,17 @@ module.exports = kind(
 	initComponents: function () {
 		this.createChrome(this.tools);
 		Popup.prototype.initComponents.apply(this, arguments);
+		this.syncProperties();
+	},
+
+	/**
+	* Sync spotlightModal and modal properties on creation time.
+	*
+	* @private
+	*/
+	syncProperties: function () {
```

Comment:
Not sure we need a separate method for this anymore, after pruning some of its operations. Could we move the assignment operation into our initialization routine, instead, and drop the early return code?